test/lit.cfg

```diff
@@ -813,6 +814,9 @@ elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'wi
       config.target_object_format = "elf"
       config.target_dylib_extension = "so"
       config.target_sdk_name = "linux"
+      if os.path.exists(make_path(config.swift_lib_dir, "swift", "linux", "libdispatch.so")):
```

Comment:
Do we ever test with the static variant of the runtime?  We may have those enabled, so we need to check both :-(.